### ISSUE TITLE
Can we run the code with more than 3 processors ?

### ISSUE THREAD

**Issue by siddharth9901** [mackncheesiest/heft#21]
I tried changing the input data by changing the number of processors to 4 and other required changes in the following files:
'../test/canonicalgraph_task_exe_time.csv'
'../test/canonicalgraph_resource_BW.csv'
'../test/canonicalgraph_task_connectivity.csv'

But getting this error: 
![Screenshot 2023-06-08 at 6 30 22 PM](https://github.com/mackncheesiest/heft/assets/58816993/3b51be50-65b0-4460-aa5d-4b93cd7df92f)

Could you please guide me to resolve this, i need this ASAP to complete my college project.

Awaiting your timely response. Thanks a lot :)

**Comment by mackncheesiest** [comment]
Yeah, the code is scalable for more processors. It would be nice to know which matrix is being indexed out of bounds on that line, but I'm assuming it's the `_self.communication_startup` matrix (the list of "startup" communication costs). 

You can pass that in with the `communication_startup` argument for [heft.schedule_dag](https://github.com/mackncheesiest/heft/blob/master/heft/heft.py#L66). That was added after this jupyter notebook was created via #14 , so it's not specified on that jupyter cell

**Comment by JonaHesselmann** [comment]
Is there an update on this issue. Having the same Problem with scaling. Tried to pass the communication_startup as an argument but got an Value Error back. Created the Graphs with the included generator. It looks like more than 4 processors triggers an array out of bounds. EDIT: Found the problem. it really was the communication_startup. I altered the default size and it now works.
